ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -89,61 +87,76 @@ internal constructor(
    * List of [LocationOfInterest] for the active survey that are present within the map bounds and
    * zoom level is clustering threshold or higher.
    */
-  val loisWithinMapBoundsAtVisibleZoomLevel: LiveData<List<LocationOfInterest>>
+  private val _loisWithinMapBoundsAtVisibleZoomLevel: MutableStateFlow<List<LocationOfInterest>> =
```

Comment:
s/...WithinMapBoundsAtVisibleZoomLevel/...InViewport/ here and throughout?

Reply:
Done